tests/subscriptions/test_scheduler.py

```diff
@@ -82,6 +83,19 @@ def test_simple(self) -> None:
             ],
         )
 
+    def test_simple_jittered(self) -> None:
+        state.set_config("subscription_primary_task_builder", "jittered")
```

Comment:
Do the tests reset the state? Should this be reset afterwards?

Reply:
They do reset the config as they wipe redis.